packages/firestore/e2e/issues.e2e.js

```diff
@@ -16,9 +16,26 @@
  */
 
 const COLLECTION = 'firestore';
+const { getE2eEmulatorHost } = require('@react-native-firebase/app/e2e/helpers');
+const jsFirebase = require('firebase/compat/app');
+const { wipe } = require('./helpers');
+require('firebase/compat/firestore');
+
+const testNumbers = {
+  zero: 0, // int
+  negativeZero: -0, // double
+  half: 0.5, // double
+  unsafeInt: Number.MAX_SAFE_INTEGER + 1, // double
+  nagativeUnsafeInt: Number.MIN_SAFE_INTEGER - 1, // double
+  safeInt: Number.MAX_SAFE_INTEGER, // int
+  nagativeSafeInt: Number.MIN_SAFE_INTEGER, // int
+  inf: Infinity, // double
+  negativeInf: -Infinity, // double
+  // nan: NaN, // double -- where-in queries on NaN do not work
+};
 
 describe('firestore()', function () {
-  describe(COLLECTION, function () {
+  describe('pagination', function () {
```

Comment:
```suggestion
  describe(COLLECTION, function () {
```

I don't understand this change, it seems unrelated, can it be left as is?

Reply:
Oh right, I based my changes on https://github.com/invertase/react-native-firebase/commit/f9ce428de0c584499642d1020c0a1b17ab15c6d5#diff-b7d5d6ae9890f87025278e8b804153bd92e1a0c6a6c9ab78acdf9ac2a8015855R178-R184 and that's where this change came from. I have reverted it now :)